snakesist/exist_client.py

```diff
@@ -13,8 +12,9 @@
 from lxml import etree  # type: ignore
 from requests.auth import HTTPBasicAuth
 
-from snakesist.errors import ExistAPIError
+import snakesist.delb_plugin as plugin
 
+delb.plugins.plugin_manager.register(plugin)
```

Comment:
this isn't necessary, you should rather declare your plugin like [here](https://github.com/funkyfuture/delb/blob/master/pyproject.toml#L27) in the package metadata.

Reply:
one needs to run `poetry install` after doing so.